### ISSUE TITLE
Compile Kotlin DSL gradle scripts of all modules in parallel

### ISSUE THREAD

**Issue by LouisCAD** [gradle/gradle#14962]
### Expected Behavior
All `.gradle.kts` files are compiled in parallel, maximizing the use of CPU cores.

### Current Behavior
All `.gradle.kts` files are compiled sequentially, using only one CPU core, dramatically delaying the run of the build.

### Context
I had edited dependencies of the `buildSrc` with Gradle 6.7.
I'd also have this issue on a newly cloned multi-modules project.

**Comment by eskatos** [comment]
This is a duplicate of https://github.com/gradle/gradle/issues/9225, closing
